docs/guide/creating-a-service.md

```diff
@@ -1,14 +1,45 @@
 # Creating a service
 
-`cd` into a directory of your choice and run `serverless create --name my-service --provider aws`.
-Serverless will create a skeleton for your new Serverless service inside the `my-service` directory.
+Let's create our first Serverless service!
+
+## Creating our service
+
+`cd` into a directory of your choice and run `serverless create --name my-service --provider aws` to create a service
+skeleton with the service name `my-service` on your local machine.
+
+Services are provider specific. The `--provider aws` tells Serverless to create an Amazon Web Services dependent service.
 
 Type `cd my-service` to navigate into the previously created directory.
 
-## Open the service inside an editor
+## Open the service inside your editor
+
+Let's take a closer look at the skeleton Serverless has created for us. Open up the `first-service` directory with your
+favorite editor.
+
+You'll see the following files:
+- `serverless.yaml`
+- `serverless.env.yaml`
+- `handler.js`
+
+### [`serverless.yaml`](../understanding-serverless/serverless-yaml.md)
+
+This is our core service file. You can see the name of our service, the provider and the first function inside the
+`functions` definition which points to the `handler.js` file.
+
+### [`serverless.env.yaml`](../understanding-serverless/serverless-env-yaml.md)
+
+This file includes all the environment variables Serverless uses. It automatically creates environment variables for a
+default stage called `dev` and a default region called `us-east-1`. Those variables will be referenced and used as a
+default later on for deployment.
```

Comment:
Same as above, should have a call to the more in depth guide mentioned in the text so people know how to get there.